test/DynamoCoreTests/PackageManager/PackageManagerClientTests.cs

```diff
@@ -33,7 +34,7 @@ public void SearchWithEmptyQueryReturnsAllPackagesInSortedOrder()
         [Test]
         public void ListAllReturnsAllPackages()
         {
-            var elements = ViewModel.Model.PackageManagerClient.ListAll();
+            var elements = this.CurrentDynamoModel.PackageManagerClient.ListAll();
             Assert.AreNotEqual(0, elements.Count());
             Console.WriteLine(elements.Count());
         }
```

Comment:
`PackageManagerClientTests` will be done in a separate PR.  It is now thoroughly mocked.